utils/vptr/README.adoc

```diff
@@ -0,0 +1,99 @@
+Software Managed Virtual Pointer
+================================
+
+This folder contains the implementation of a software-managed virtual
+pointer that facilitates a pointer interface for SYCL buffers.
+This virtual address
+space is non-dereferenceable on the host, but accessors can be obtained
+in the SYCL command groups.
+
+This differs from the legacy_pointer interface in that a contiguous
```

Comment:
It is an alternative implementation, a little bit more complicated but with less limitations in terms of indexing.